src/store/RelayRecordStore.js

```diff
@@ -112,10 +113,11 @@ class RelayRecordStore {
   /**
    * Get the data ID associated with a storage key (and optionally an
    * identifying argument value) for a root query.
+   * toDO ?any should be something like number|string|object
    */
   getDataID(
     storageKey: string,
-    identifyingArgValue: ?string
+    identifyingArgValue: any
```

Comment:
Again, please do not use `any` as a type. As I've mentioned on this PR before, this bypasses Flow type checking and reduces our chances of catching edge case bugs just like the one in the previous comment.